src/tests/functional/shared_test_classes/src/base/utils/compare_results.cpp

```diff
@@ -3,6 +3,8 @@
 //
 
 #include "ngraph/ops.hpp"
+#include "ngraph_ops/augru_cell.hpp"
+#include "ngraph_ops/augru_sequence.hpp"
```

Comment:
Are those 2 includes needed after introducing the `#include "ngraph_ops/opset_private_tbl.hpp"`?

Reply:
yes, they need. Please check opset_private_tbl.hpp which is alike opset9_tbl.hpp.
Maybe we still need another ops_private.hpp to minic ops.hpp to include the two private headers. I can add one in this PR or left it over until more private ops invented.

Reply:
Ok, understood. We can leave those 2 includes as is for now.